src/dropdown/dropdown.component.ts

```diff
@@ -328,6 +328,9 @@ export class Dropdown implements OnInit, AfterContentInit, OnDestroy, ControlVal
 			// only emit selected for "organic" selections
 			if (event && !event.isUpdate) {
 				this.selected.emit(event);
+			} else {
+				// emit an empty array when the last selected item in a multiselect is unselected
+				this.selected.emit([]);
```

Comment:
This is going to end up emitting an empty array every time `select` emits, which could be quite a lot - `isUpdate` exists to let us switch between updates for programmatic reasons and updates due to user selection, we only want to emit `selected` when it's a user selection